[21-Nov-2021 13:37:49 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:37:57 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:38:12 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:38:20 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:38:23 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:38:51 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:38:58 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('js/limtBox.js')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:56:18 Europe/Warsaw] Uncaught exception: 'ErrorException'with message 'Undefined property: App\Models\Balance::$incomeCategoryRecords'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php(54): Core\Error::errorHandler(2, 'Undefined prope...', 'G:\\programy\\XAM...', 54)
#1 [internal function]: App\Controllers\BalanceManager->addAction()
#2 G:\programy\XAMPP\htdocs\phpMVC\Core\Controller.php(23): call_user_func_array(Array, Array)
#3 G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php(134): Core\Controller->__call('add', Array)
#4 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('BalanceManager/...')
#5 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php' on line 54
[21-Nov-2021 13:56:18 Europe/Warsaw] Uncaught exception: 'ErrorException'with message 'Undefined property: App\Models\Balance::$incomeCategoryRecords'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php(54): Core\Error::errorHandler(2, 'Undefined prope...', 'G:\\programy\\XAM...', 54)
#1 [internal function]: App\Controllers\BalanceManager->addAction()
#2 G:\programy\XAMPP\htdocs\phpMVC\Core\Controller.php(23): call_user_func_array(Array, Array)
#3 G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php(134): Core\Controller->__call('add', Array)
#4 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('BalanceManager/...')
#5 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php' on line 54
[21-Nov-2021 13:56:59 Europe/Berlin] Uncaught exception: 'Exception'with message 'No route matched.'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php' on line 146
[21-Nov-2021 13:57:37 Europe/Berlin] Uncaught exception: 'ErrorException'with message 'Undefined property: App\Models\Balance::$startDate'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php(229): Core\Error::errorHandler(2, 'Undefined prope...', 'G:\\programy\\XAM...', 229)
#1 G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php(314): App\Models\Balance->setExpenseCategory()
#2 G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php(212): App\Models\Balance->setExpenses()
#3 G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php(41): App\Models\Balance->save()
#4 [internal function]: App\Controllers\BalanceManager->addAction()
#5 G:\programy\XAMPP\htdocs\phpMVC\Core\Controller.php(23): call_user_func_array(Array, Array)
#6 G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php(134): Core\Controller->__call('add', Array)
#7 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('BalanceManager/...')
#8 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php' on line 229
[21-Nov-2021 13:57:57 Europe/Warsaw] Uncaught exception: 'ErrorException'with message 'Undefined property: App\Models\Balance::$sumOfIncomes'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php(213): Core\Error::errorHandler(2, 'Undefined prope...', 'G:\\programy\\XAM...', 213)
#1 G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php(41): App\Models\Balance->save()
#2 [internal function]: App\Controllers\BalanceManager->addAction()
#3 G:\programy\XAMPP\htdocs\phpMVC\Core\Controller.php(23): call_user_func_array(Array, Array)
#4 G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php(134): Core\Controller->__call('add', Array)
#5 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('BalanceManager/...')
#6 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php' on line 213
[21-Nov-2021 13:58:04 Europe/Berlin] Uncaught exception: 'Error'with message 'Call to undefined method App\Models\Balance::setIncoesCategory()'
Stacktrace: #0 G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php(211): App\Models\Balance->setIncomes()
#1 G:\programy\XAMPP\htdocs\phpMVC\App\Controllers\BalanceManager.php(41): App\Models\Balance->save()
#2 [internal function]: App\Controllers\BalanceManager->addAction()
#3 G:\programy\XAMPP\htdocs\phpMVC\Core\Controller.php(23): call_user_func_array(Array, Array)
#4 G:\programy\XAMPP\htdocs\phpMVC\Core\Router.php(134): Core\Controller->__call('add', Array)
#5 G:\programy\XAMPP\htdocs\phpMVC\public\index.php(72): Core\Router->dispatch('BalanceManager/...')
#6 {main}
Thrown in 'G:\programy\XAMPP\htdocs\phpMVC\App\Models\Balance.php' on line 185
